scripts/add_translation.py

```diff
@@ -71,18 +80,47 @@ def get_credentials():
     return credentials
 
 
+def display_languages(service):
+    print 'List of available languages:\n----------------------------'
+    data = service.files().list(q="'" + TRANSLATION_FOLDER_ID + "'" + " in parents").execute()
+    for f in data['files']:
+        if f['mimeType'].endswith('.folder'):
+            LANGUAGE_LIST[f['name']] = f['id']
+
+    # sort the languages alphabetically
+    sorted_list = [e for e in sorted(LANGUAGE_LIST)]
+    id = 1
+    # store the generated IDs to match against user input.
+    language_id = {}
+    for i in sorted_list:
+        print  str(id) + ' ' + i
+        language_id[id] = i
+        id = id + 1
+
+    response = raw_input('Select language from the list: ')
+    user_choice = language_id[int(response)]
+    print 'User selected '+user_choice+ ' language'
+    if user_choice not in LANGUAGE_LIST:
+        print '\nInvalid selection'
+        exit(1)
```

Comment:
Use `sys.exit` instead of `exit` for all exit calls.

To be honest, this is one of the more needlessly confusing things in python, so here's a useful reference explaining in more detail:
http://stackoverflow.com/questions/19747371/python-exit-commands-why-so-many-and-when-should-each-be-used